com.playeveryware.eos/CHANGELOG.md

```diff
@@ -4,6 +4,282 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Changelog
 
+## [3.3.2] - 2024-08-27
+
+### Added
+
+- Git Actions
+  - feat: Create check-for-package-difference.yml
```

Comment:
An argument could be made that this package difference feat doesn't need to be in here because it doesn't apply to the plugin itself.